packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -131,9 +142,12 @@ export class Telemetry {
       Telemetry.options.setupString,
     );
 
-    // Uncomment for Fiddler testing
-    // Telemetry.client.config.proxyHttpUrl = 'http://localhost:8888';
-    // Telemetry.client.config.proxyHttpsUrl = 'http://localhost:8888';
+    // Allow overriding the proxy server via an environment variable
+    const proxyServer = process.env[ENV_PROXY_OVERRIDE];
+    if (proxyServer) {
+      Telemetry.client.config.proxyHttpUrl = proxyServer;
+      Telemetry.client.config.proxyHttpsUrl = proxyServer;
```

Comment:
do we need separate http/https endpoints?

Reply:
Technically I don't expect anyone to need this but us for testing purposes. In practice, you're going to set both to the same value anyway.

For people who need to specify proxies for real, the library already defaults to looking at the `http_proxy` and `https_proxy` environment variables, which is a standard practice for node libraries.

The only reason I'm creating a separate override here is because I need to test just redirecting the telemetry traffic through Fiddler, not all of node, because packagers don't like calling through Fiddler.